pynitrokey/nk3/otp_app.py

```diff
@@ -33,6 +33,43 @@ class SelectResponse:
     algorithm: Optional[bytes]
 
 
+@dataclasses.dataclass
+class OTPAppException(Exception):
+    code: str
```

Comment:
We could add an ISO 7816 `Status` enum for this field.  This would also help in other functions like https://github.com/Nitrokey/pynitrokey/blob/master/pynitrokey/cli/nk3/test.py#L206.

Reply:
Sure, can be done. I cut the corner here, thinking this would not be reusable. I wonder if there a Python library with the error codes already (`pyscard` ?).

Reply:
To be done later. Registered as https://github.com/Nitrokey/pynitrokey/issues/297.